### DISCUSSION TITLE
Grid System Why are the sizes different?

### DISCUSSION THREAD

**Discussion by JensBond** [tailwindlabs/tailwindcss#2863] [Help] [answered]
Hi,

I just played arround with the new CSS Grid Sytem. 

Can anyone tell my why the label from div Four is longer than the other ones?
What is the best way to fix thoes issues?
https://play.tailwindcss.com/IqYsmfxpwJ

Thanks

**Comment by MartinHughes-BPC** [comment, marked as answer]
Yes - you spelled label incorrectly!

**Reply by JensBond** [reply]
Basically you are right... but fortgoss example you can use any tag and it wount work

**Comment by JensBond** [comment]
I think the problem is the gap which is missing between this boxe. That has the effect that the box is 4 pixel bigger. 

But how can I fix that the best way? Should I work with a direct pixel width for the label?

**Reply by MartinHughes-BPC** [reply]
Ah, I see what you mean. I think what's happening is that the gap is causing the other labels to shrink - this is very noticable when you set a big gap (like 10).

I'll have a play around to see if I can come up with anything.

**Reply by JensBond** [reply]
Yes I now use in the grid flex boxes to give the label a fix width.
